cmd/gaiad/cmd/root.go

```diff
@@ -83,7 +83,7 @@ func initAppConfig() (string, interface{}) {
 
 	return params.CustomConfigTemplate(), params.CustomAppConfig{
 		Config:               *srvCfg,
-		BypassMinFeeMsgTypes: gaia.GetDefaultBypassFeeMessages(),
+		BypassMinFeeMsgTypes: []string{},
```

Comment:
I think we still want set default bypass msg when do `gaiad init` ?

Reply:
The default with this change means that no bypass messages will be set for you. You would need to change the config yourself.

Do you think it shouls be pre-populated?

Reply:
it is pre-populated, and nodes can change it.

Reply:
Done!